### ISSUE TITLE
Regex.IsMatch(String input) throws ArgumentNullException inside DebugProxyLauncher 

### ISSUE THREAD

**Issue by miroslavp** [dotnet/aspnetcore#20238]
### Describe the bug
I've upgraded my **Blazor Webassembly** project from **3.2 Preview 2** to **Preview 3** and if I put 
`"inspectUri": "{wsProtocol}://{url.hostname}:{url.port}/_framework/debug/ws-proxy?browser={browserInspectUri}"`
 inside **launchSettings.json** file I get this exception from my **Visual Studio Community 2019 Preview Version 16.6.0 Preview 2.0** when I run the app

`System.ArgumentNullException
  HResult=0x80004003
  Message=Value cannot be null. (Parameter 'input')
  Source=System.Text.RegularExpressions
  StackTrace:
   at System.Text.RegularExpressions.Regex.IsMatch(String input)
   at Microsoft.AspNetCore.Builder.DebugProxyLauncher.c__DisplayClass9_0.g__OnOutputDataReceived|0(Object sender, DataReceivedEventArgs eventArgs)
   at System.Diagnostics.Process.OutputReadNotifyUser(String data)
   at System.Diagnostics.AsyncStreamReader.FlushMessageQueue(Boolean rethrowInNewThread)
   at System.Runtime.ExceptionServices.ExceptionDispatchInfo.Throw()
   at System.Diagnostics.AsyncStreamReader.c.b__18_0(Object edi)
   at System.Threading.QueueUserWorkItemCallback.c.b__6_0(QueueUserWorkItemCallback quwi)
   at System.Threading.ExecutionContext.RunForThreadPoolUnsafe[TState](ExecutionContext executionContext, Action 1 callback, TState& state)
   at System.Threading.QueueUserWorkItemCallback.Execute()
   at System.Threading.ThreadPoolWorkQueue.Dispatch()
   at System.Threading._ThreadPoolWaitCallback.PerformWaitCallback()`

`This exception was originally thrown at this call stack:
    [External Code]`

What am I doing wrong? What is this 'input' parameter and where should I provide it?

**Comment by miroslavp** [comment]
So, the exception above is copied from VisualStudio error dialog box. I also found another exception that is logged in the EventLog. Here it is:

Application: dotnet.exe
CoreCLR Version: 4.700.20.11803
.NET Core Version: 3.1.3
Description: The process was terminated due to an unhandled exception.
Exception Info: Internal.Cryptography.CryptoThrowHelper+WindowsCryptographicException: The filename, directory name, or volume label syntax is incorrect.
   at Internal.Cryptography.Pal.CertificatePal.FromBlobOrFile(Byte[] rawData, String fileName, SafePasswordHandle password, X509KeyStorageFlags keyStorageFlags)
   at System.Security.Cryptography.X509Certificates.X509Certificate..ctor(String fileName, String password, X509KeyStorageFlags keyStorageFlags)
   at System.Security.Cryptography.X509Certificates.X509Certificate2..ctor(String fileName, String password)
   at Microsoft.AspNetCore.Server.Kestrel.KestrelConfigurationLoader.LoadCertificate(CertificateConfig certInfo, String endpointName)
   at Microsoft.AspNetCore.Server.Kestrel.KestrelConfigurationLoader.Load()
   at Microsoft.AspNetCore.Server.Kestrel.Core.KestrelServer.ValidateOptions()
   at Microsoft.AspNetCore.Server.Kestrel.Core.KestrelServer.StartAsync[TContext](IHttpApplication`1 application, CancellationToken cancellationToken)
   at Microsoft.AspNetCore.Hosting.GenericWebHostService.StartAsync(CancellationToken cancellationToken)
   at Microsoft.Extensions.Hosting.Internal.Host.StartAsync(CancellationToken cancellationToken)
   at Microsoft.Extensions.Hosting.HostingAbstractionsHostExtensions.RunAsync(IHost host, CancellationToken token)
   at Microsoft.Extensions.Hosting.HostingAbstractionsHostExtensions.RunAsync(IHost host, CancellationToken token)
   at Microsoft.Extensions.Hosting.HostingAbstractionsHostExtensions.Run(IHost host)
   at Microsoft.AspNetCore.Components.WebAssembly.DebugProxy.Program.<>c__DisplayClass0_0.<Main>b__0() in /_/src/Components/WebAssembly/DebugProxy/src/Program.cs:line 76
   at Microsoft.Extensions.CommandLineUtils.CommandLineApplication.Execute(String[] args) in /_/.packages/microsoft.extensions.commandlineutils.sources/3.1.2-servicing.20067.6/contentFiles/cs/netstandard1.0/CommandLine/CommandLineApplication.cs:line 328
   at Microsoft.AspNetCore.Components.WebAssembly.DebugProxy.Program.Main(String[] args) in /_/src/Components/WebAssembly/DebugProxy/src/Program.cs:line 81

**Comment by miroslavp** [comment]
Ok, fixed my issue. 
I commented out the urls line in appsettings.json (in the Server project)
`// "Urls": "http://localhost:60422;http://192.168.0.107:60422"`

The second exception was because of wrongly configured ssl in Kestrel section (which I don't use for now anyway). 

After commenting out both the Urls and Kestrel sections, the errors disappeared and I was able successfully to run and debug the app. If I leave either of those uncommented, the error persists.
